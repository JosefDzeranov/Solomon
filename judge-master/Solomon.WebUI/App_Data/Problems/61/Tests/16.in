77 38
ammo
shoot
save
load
ammo
save
save
shoot
shoot
save
shoot
load
save
shoot
ammo
ammo
load
save
ammo
ammo
ammo
load
ammo
shoot
ammo
load
shoot
ammo
ammo
shoot
load
ammo
save
save
shoot
load
shoot
save
save
load
shoot
shoot
ammo
ammo
load
load
save
shoot
shoot
save
ammo
shoot
load
ammo
shoot
load
save
load
save
ammo
ammo
ammo
shoot
load
ammo
save
save
save
shoot
ammo
save
save
save
shoot
ammo
shoot
shoot
