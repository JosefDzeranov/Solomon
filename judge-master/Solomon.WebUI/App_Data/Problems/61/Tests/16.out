38
37
37
37
75
75
75
74
73
73
72
73
73
72
110
148
73
73
111
149
187
73
111
110
148
73
72
110
148
147
73
111
111
111
110
111
110
110
110
110
109
108
146
184
110
110
110
109
108
108
146
145
108
146
145
108
108
108
108
146
184
222
221
108
146
146
146
146
145
183
183
183
183
182
220
219
218
